postgres_scanner.cpp

```diff
@@ -939,7 +942,7 @@ AND table_type='BASE TABLE'
 		dconn
 		    .TableFunction(data.filter_pushdown ? "postgres_scan_pushdown" : "postgres_scan",
 		                   {Value(data.dsn), Value(data.source_schema), Value(table_name)})
-		    ->CreateView(table_name, data.overwrite, false);
+		    ->CreateView(data.sink_schema,table_name, data.overwrite, false);
```

Comment:
Can you fix the formatting here please? You should just be able to run `make format`

Reply:
got it

Reply:
done.

Reply:
@Mause any chance to merge PR?